Remotion/ObjectBinding/Web/UI/Controls/BocEnumValue.cs

```diff
@@ -388,7 +395,7 @@ protected void EnsureValue ()
       else if (_internalValue != null && Property != null)
       {
         //  Can get a new EnumerationValueInfo
-        _enumerationValueInfo = Property.GetValueInfoByIdentifier (_internalValue, GetBusinessObject());
+        _enumerationValueInfo = Property.GetValueInfoByIdentifier (_internalValue, GetBusinessObject()!)!;
```

Comment:
```suggestion
        _enumerationValueInfo = Property.GetValueInfoByIdentifier (_internalValue, GetBusinessObject());
```

Reply:
`GetValueInfoByIdentifier` might return `null`. How should this be handled? Assertion in the following line?

Reply:
yeah, when the internalValue is not null, the result should be null either. => assertion.IsNotNull with a reference to the actual string value that resulted in null.

Reply:
This means I could also add `[return: NotNullIfNotNull ("identifier")]` to the method declaration, right?

Reply:
nevermind, the callchain beneath is too complex - I'll add the assertion